workflow-templates/mendix-actions.yml

```diff
@@ -0,0 +1,43 @@
+## Ensure that all lines with comments are filled out correctly ##
+
+name: Actions
+
+on:
+  push:
+    branches:
+      - "development"
+      - "test"
+      - "main"
+  pull_request:
+    branches:
+      - "development"
```

Comment:
Should this section use the [`$default-branch`](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization#creating-a-starter-workflow) token instead of explicit branch names or do we want to be explicit here?

Reply:
Should my setup documentation be enough to ensure the branch names are the same?